src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -16,6 +16,7 @@ namespace Microsoft.DotNet.Cli.Utils
     internal class MSBuildForwardingAppWithoutLogging
     {
         private static readonly bool AlwaysExecuteMSBuildOutOfProc = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_RUN_MSBUILD_OUTOFPROC");
+        private static readonly bool DoNotUseMSBUILDNOINPROCNODE = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_DO_NOT_USE_MSBUILDNOINPROCNODE");
```

Comment:
nit: Maybe remove the double negation: `DOTNET_CLI_USE_MSBUILD_INPROC_NODE` ?

Reply:
I was trying to keep name same as is the MSBUILD escape hatch toi avoid confusion between your OUTOFPROC and mine NOINPROC.  People might get impression that OUTOFPROC == NOINPROC and get confused why we have two escapes for same thing. Ithough about `DOTNET_CLI_DONT_USE_MSBUILD_SERVER_V1` but I believe it is confusing for anyone without knowledge of msbuild server background....